### DISCUSSION TITLE
How can I write a price like 35.000,33 Euro?

### DISCUSSION THREAD

**Discussion by EugenioFe** [baimard/gestion#366] [Q&A]
I am having problems to write the product price. I am trying to write something like 35.000,33 Euro.
However the system allows me only 35 euros... When I write 35000 or 35.000, I become an error message.
Thanks for your support,
Eugenio

**Comment by baimard** [comment]
Test with 35000.33 if you're in En-en

**Reply by EugenioFe** [reply]
First of all, thanks for your APP. My respect for you.
I find it very interesting and I am trying to integrate it in my start-up.

Yes, I am with en-En in your APP. 

I follow your suggestion.
Ok, I think I have caught what happens.
When I write your proposal + enter, I receive a failure message and €NaN appears.
When I go to another menu and go back to "products", it appears: "Modification saved" + "there is an error with format. Please, check documentation". And the correct figure appears: €35,000.00 


![imagen](https://github.com/baimard/gestion/assets/117111064/ec4ebb24-188f-4536-a44e-c409ea9c8464)
![imagen](https://github.com/baimard/gestion/assets/117111064/8effce39-fcf1-4270-90fc-10a7ae167838)

**Reply by baimard** [reply]
Yes I Identify a bug, when you first try to change a price NaN appear don't know why. I will investigate.

thank you !

**Reply by EugenioFe** [reply]
Merci
